backend/src/mappers/AppUserMapper.ts

```diff
@@ -18,15 +19,27 @@ export class AppUserMapper {
    * Converts an EventSignInRequest payload to an AppUser entity and
    * returns the newly created entity to the caller.
    *
-   * @param {AppUserPostRequest | AppUserEventRequest} appUserRequest The request payload from which the AppUser entity is created.
+   * @param {AppUserPostRequest | AppUserEventRequest | InducteePostRequest} appUserRequest The request payload from which the AppUser entity is created.
    * @returns {AppUser} A newly created AppUser entity.
    */
   async requestToNewEntity(
-    appUserRequest: AppUserPostRequest | AppUserEventRequest
+    appUserRequest: AppUserPostRequest | AppUserEventRequest | InducteePostRequest
   ): Promise<AppUser> {
     const appUserRepository = getRepository(AppUser);
     const inductionClassRepository = getRepository(InductionClass);
 
+    if (appUserRequest instanceof InducteePostRequest) {
```

Comment:
Since this if block contains the same content as the one below it, you can just combine the conditionals i.e. `appUserRequest instanceof InducteePostRequest || appUserRequest instanceof AppUserPostRequest` or something like that